### DISCUSSION TITLE
Welcome to Thorium Discussions!

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium#1] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with Thorium and Chromium users/devs. Here you can:
  * Ask ANY/ALL questions you’re wondering about. (It doesn't even have to concern Thorium.)
  * Share ideas. (FOSS, Linux, and Thorium ideas are especially welcome.)
  * Engage with other community members.
  * Be respectful and open-minded. Remember that we build the GitHub community and the software it hosts together 💪.
  * Ask about future plans and ask to contribute/make pull requests.

  To get started, comment below with an introduction of yourself and tell us about what you do within the scope of this community or the Thorium browser.

<img src="https://raw.githubusercontent.com/Alex313031/Thorium/main/logos/STAGING/error_dog.png" width="256">

**Comment by pierro78** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do within the scope of this community or the Thorium browser.

Hey thanks for building a community here !
Right now I am interested in small, light and "sturdy" laptops for web browsing (even made a list : https://docs.google.com/spreadsheets/d/1-8ISd9k6u9OhkDbdLX7PcLN82Kd39y31etZeVT8BTSY/edit?usp=sharing )
I find that Chrome(ium)OS has the best UI for web browsing but chromebooks are limited compared to regular PCs which can also run Windows or Linux natively. The advantage of Windows is that it's well supported but it's not so lightweight compared to Chrome(ium)OS or Linux.

Right now I am fighting with a small Chuwi HeroBook Air (got it for 174Euros on 11/11) ... tried ChromeOS (via brunch) on it but the wifi would only work once ... on linux there is a wifi driver for it (also used by brunch) but then there is an issue with audio ( https://forum.chuwi.com/t/linux-on-chuwi-gemibook/24770 : I even compiled the 5.16git kernel merged with newest Sound Open Framework to try to make audio work but I am not knowledgeable enough and this breaks several things ...) ... Windows is OK but sometimes a little slow on this weakish machine (but antix linux runs really nicely on it)  ... but maybe a special Thorium build for celeron n4020 (SSE4.2) would make using Windows a little faster ;)

**Reply by Alex313031** [reply]
I am looking into cross building for windows on my linux machine. That way I don't have to boot into windows and do everything seperately. I followed the chromium guide for cross building, but got stuck on one part. I am talking with robrich about how he does it as all his win builds are done on his linux server. Robrich taught me alot about compiler optimizations and some of his suggestions made it into thorium. We have a nice lil discussion group going on that me and him frequent alot, and I think it is something you might wanna join or just stay up to date on so here it is. > https://github.com/RobRich999/Chromium_Clang/issues/26

**Comment by badbucket1301** [comment]
Hi and thank you for providing custom chromium builds and trying to build a community around them!
Especially now that robrich has closed his [repo](https://github.com/RobRich999/Chromium_Clang)! :(

I am a hardware nerd and scriptkid with a passion for data privacy and can offer to help test new features or ideas.
Being able to give feedback and be involved in this project is awesome!

Still using his latest build but will switch to the latest Thorium release after posting this.
Found you linked on woolyss' page after trying to find a new source for builds.

More on my use case, I am using modified chromium builds mainly for the added codecs and resulting compatibility on the web.
A Pi-Hole, a zillion launch args and two dozen extensions are taking care of my privacy needs. Though I do very much appreciate the more privacy-focused forks like bromite that I am running on my phone.

The devices I use are a 9700K Win11 pc, a 7700K manjaro server, a OnePlus 7 Pro and a 6700HQ Win11 Notebook.

Looking forward to everything that's to come with Thorium.
Cheers

**Reply by Alex313031** [reply]
Have you tried it yet?

**Comment by Alex313031** [comment]
@einsnulleins Man with a 9700k you could compile thorium in like 2 hours or less. Im on a 4790K OC'd to 4.6Ghz, and it takes ~4 hours

**Comment by paulindr** [comment]
Just interested in trying new browsers, not just staying with one out of habit. 
Firefox suits me but Thorium looks good (if it didn't prent me with a problem straight away!)

**Comment by berlintay** [comment]
This really took me by surprise! What an amazing browser this is. I've been through every browser the past months and when i landed on Thorium, I was shocked to see how fast everything is, and the design is immaculate. I like the three.js features, im not an expert on web design; but i could imagine having extended threejs themes, such as customized home pages, animated tabs (glowing, sparkling stars.. Etc.)
Well done. If you need any contribution to docs, or whatsover please reach out!

**Comment by Lom-666** [comment]
hello everyone. I don't know exactly where to write, so I decided to write here. I started using the Thorium browser for the first time, starting with the penultimate version. at the moment, I have this latest version installed - https://i.imgur.com/3qJtYeu.png
the fact is that when watching videos in non-fullscreen mode, different artifacts appear in the browser from time to time, of different sizes in the likeness of this https://i.imgur.com/6MeDn7k.png (there are no similar problems in other browsers and never have been, and this is the first time I see this, only in Thorium browser) tried disabling all extensions, but it didn't help.
what could be the problem in your opinion? my system's data - https://i.imgur.com/e3xBkes.png and https://i.imgur.com/4w5SEQP.png
**And the last thing. Thank you so much for such a fast browser!!!**

**Reply by gz83** [reply]
You can post your screenshot directly in GitHub without using an external link image.

In addition, have you updated all graphics card drivers to the latest version? Are there any experimental flags turned on (can be seen in chrome://flags)?

@Lom-666

**Reply by Lom-666** [reply]
yes, I updated the drivers, because I thought maybe it was a problem with them, but it also didn't help.  about the flags. I didn't turn anything on, but only turned off the two "Hide Sidebar button" and "Chrome Labs", but the problem was even BEFORE disabling these two flags.

**Reply by gz83** [reply]
Can you send me the exact URL of the video in question? I can't get any more information from just the pictures.

@Lom-666

**Reply by Lom-666** [reply]
it doesn't matter which video. any video from YouTube or from another site where there are movies. the problem arises precisely when you are viewing in non-fullscreen mode.

**Reply by gz83** [reply]
I just ran some tests and found no artifacts as you mentioned.

@Lom-666

**Reply by Lom-666** [reply]
understood. anyway, thanks for answering me))

**Reply by Lom-666** [reply]
> Я только что провел несколько тестов и не обнаружил никаких артефактов, как вы упомянули.
> 
> @Lom-666

Hi. is it possible to add here https://i.imgur.com/DhUa7Zp.png to show the download speed?

**Comment by Kales4** [comment]
I'm here just looking for a browser that still works for my old W7 64bit.

So I installed a Thorium browser for W7. My thorium browser is V 119.0.6045.214. 

To be completely honest it is the worst browser I've ever installed. Sorry, but it crashes 10 to 15 times per minute. Also it can't be updated from within. What this means is that if I find a link for the latest version, unless proper instructions, it will override everything.

Can someone please just give me a link to the latest version of this browser. 

Thanks in advance anyway

**Reply by gz83** [reply]
https://github.com/Alex313031/thorium-legacy/releases

You posted a comment in the wrong repository

@Kales4

**Reply by Kales4** [reply]
I'm very sorry (and new here), but thanks if your link directs me to the right place. I'll check it later  ...
